### PR TITLE
Add permissions just for logged in users

### PR DESCRIPTION
**Pull request by annagav**
#### What are the relevant tickets?
Fix https://trello.com/c/saaYywwv/37-restrict-access-to-logged-in-users

#### What's this PR do?
Adds a video view permission that allows all logged in users to view the video.

#### How should this be manually tested?
Screen Shot 2020-05-04 at 9 14 07 AM

Screen Shot 2020-05-05 at 2 29 54 PM

### CODE REVIEW

**Comment by Ferdi** [comment]
Can you replace "only logged in" with "MIT Touchstone"

File: static/js/components/dialogs/CollectionFormDialog_test.js [hunk 1]
```diff
@@ -197,11 +197,12 @@ describe("CollectionFormDialog", () => {
...
-          description:   "new description",
-          view_lists:    expectedListRequestData,
-          admin_lists:   expectedListRequestData,
-          edx_course_id: "edx-course-id"
+          title:             "new title",
+          description:       "new description",
+          view_lists:        expectedListRequestData,
+          admin_lists:       expectedListRequestData,
+          edx_course_id:     "edx-course-id",
+          is_logged_in_only: false
```
Comment (mbertrand) [inline static/js/components/dialogs/CollectionFormDialog_test.js:205]: Can you add another set of conditions to[ this test here](https://github.com/mitodl/odl-video-service/blob/4bfc90a169638463f323950023cf4e24e114232b/static/js/components/dialogs/CollectionFormDialog_test.js#L99):
```javascript
        [
          "#view-perms-only-logged-in-users",
          "viewChoice",
          SET_VIEW_CHOICE,
          PERM_CHOICE_LOGGED_IN
        ],
```

File: static/js/components/dialogs/EditVideoFormDialog.js [hunk 2]
```diff
@@ -255,6 +260,16 @@ class EditVideoFormDialog extends React.Component<*, void> {
               validationMessage={errors ? errors.view_lists : ""}
             />
           </Radio>
+          <Radio
+            id="view-only-logged-in-users"
```
Comment (mbertrand) [inline static/js/components/dialogs/EditVideoFormDialog.js:264]: On the collection form, the id is `view-logged-in-only` - can you make them the same on both (either one is fine)?

**Comment by annagav** [comment]
@mbertrand I addressed your comments. Thank you!

**Comment by pdpinch** [comment]
@annagav can you update the screenshots?

**Comment by annagav** [comment]
@Ferdi @pdpinch Here are the updated screen shots:
Screen Shot 2020-05-06 at 1 31 45 PM
Video edit:
Screen Shot 2020-05-06 at 1 43 29 PM

**Pull request merged**